src/components/TrendingCard/__tests__/TrendingCard.spec.js

```diff
@@ -19,4 +19,20 @@ describe('TrendingCard', () => {
       render(<TrendingCard element={Link} />).container.firstChild
     ).toMatchSnapshot();
   });
+
+  describe('automated accessibility testing', () => {
+    it('should have no Axe violations', async () => {
+      const { container } = render(<TrendingCard />);
+      await expect(container).toHaveNoAxeViolations();
+    });
+
+    it('should have no DAP violations', async () => {
+      const { container } = render(
+        <main>
+          <TrendingCard />
+        </main>
```

Comment:
Context: DAP tests should be wrapped in a landmark element, otherwise they will throw a violation for orphaned content.

Reply:
Thanks for the context! If we always know that we'll need to wrap these in a `main`, would it be worth including that as part of the `toHaveNoDAPViolations` matcher?

Reply:
I thought about that but there will likely be instances where we are testing a component that is already inside of a landmark element, so wrapping everything in a `main` may cause issues. I think it's better to adjust per component since it's such a small thing (plus if you forgot to wrap in a landmark element when you were supposed to, it's a good way to catch if you forgot to check the tests? 😅 )

Reply:
That sounds good! Let's run with this for now and we can tweak later down the line if needed